src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
+    }
+
     //this.checkOverflow()
   }
 
+  componentWillReceiveProps(nextProps) {
+    const { start, scrollTop } = this.props;
+    // When paginating, reset scroll
+    if (!dates.eq(nextProps.start, start) || nextProps.scrollTop !== scrollTop) {
```

Comment:
`dates.eq(nextProps.start, start, 'day')` and probably the same for: `nextProps.scrollTop !== scrollTop` no? Well not `'day'` but perhaps `'minutes'`, ie only recalculate with the scrollToTime has changed more than a minute.

Also `scrollTop` is wrong here right?